### ISSUE TITLE
"options" should be optional when running "vrtTrack"

### ISSUE THREAD

**Issue by ngraf** [Visual-Regression-Tracker/agent-codeceptjs#27]
https://github.com/Visual-Regression-Tracker/agent-codeceptjs/blob/3891780ff922b7c21cb998f6fba699b1c8295a89/lib/index.ts#L41

Why "options" is mandatory?
Please make it optional, so that Typescript linting is not complaining.
Why I have this expectation? Because in the `README.md` an example is given where no option is passed to `vrtTrack` and because I personally also think it makes sense to make it optional when there is no reason to make it mandatory.

**Comment by pashidlos** [comment]
@ngraf good point!
would you like to contribute with the fix?

**Comment by ngraf** [comment]
No, sorry. I am not a good developer. I am just a passionate QA/user of the agent.
